source/features/index.tsx

```diff
@@ -201,9 +201,8 @@ const add = async (meta?: FeatureMeta, ...loaders: FeatureLoader[]): Promise<voi
 	}
 
 	// Register feature shortcuts
-	for (const hotkey of Object.keys(shortcuts)) {
-		// TODO: use Object.entries, change format of shortcutMap
-		const description = shortcuts[hotkey];
+	for (const [hotkey, description] of Object.entries(shortcuts)) {
+		// TODO: change format of shortcutMap
 		shortcutMap.set(hotkey, {hotkey, description});
```

Comment:
I think this is the format I was referring to

```suggestion
		shortcutMap.set(hotkey, description);
```

Reply:
https://github.com/yakov116/refined-github/commit/5d2a6dc6162a103a205c1ffddd0ac297da785554

Good?